apps/client/src/common/utils/socket.ts

```diff
@@ -143,8 +143,8 @@ export const connectSocket = () => {
           break;
         }
         case 'ontime-timer': {
-          patchRuntime('timer', payload);
-          updateDevTools({ timer: payload });
+          runtimeStore.setState({ timer: payload, clock } as Partial<RuntimeStore>);
```

Comment:
using the patchRuntime provides some advantages, including us knowing that the state is up to date, and all state changes go through the same place
I wonder if it would be preferable:
- a) to make two calls
- b) modify the patch function to receive a patch object

what do you think?